<ui version="4.0" >
 <class>CGexFtpService_widget_base</class>
 <widget class="QWidget" name="CGexFtpService_widget_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ftp service</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBoxService" >
     <property name="title" >
      <string>Service</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="_2" >
        <item>
         <widget class="QPushButton" name="buttonStopService" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Stop the GEX-FTP service</string>
          </property>
          <property name="text" >
           <string>Stop service</string>
          </property>
          <property name="icon" >
           <iconset resource="gex_ftp.qrc" >
            <normaloff>:/gex-ftp/images/resources/service_stop.png</normaloff>:/gex-ftp/images/resources/service_stop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonStartService" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Start the GEX-FTP service</string>
          </property>
          <property name="text" >
           <string>Start service</string>
          </property>
          <property name="icon" >
           <iconset resource="gex_ftp.qrc" >
            <normaloff>:/gex-ftp/images/resources/service_start.png</normaloff>:/gex-ftp/images/resources/service_start.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonLoadServiceSettings" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Load settings used by the GEX-FTP service</string>
          </property>
          <property name="text" >
           <string>Load from &#xd;&#xd;&#xd;&#xd;
service settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSaveServiceSettings" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Save current settings to the file used by the GEX-FTP service</string>
          </property>
          <property name="text" >
           <string>Save to &#xd;&#xd;&#xd;&#xd;
service settings</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer18" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>92</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="_3" >
        <item>
         <widget class="QLabel" name="labelServiceHomeDir" >
          <property name="text" >
           <string>Service home directory:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editServiceHomeDir" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonBrowseServiceHomeDir" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>30</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>30</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Select the service home directory</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <spacer name="spacer19" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>16</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="labelLog" >
        <property name="text" >
         <string>Service log: ??</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QTreeWidget" name="treeWidgetLog" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="columnCount" >
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>buttonStopService</tabstop>
  <tabstop>buttonStartService</tabstop>
  <tabstop>buttonLoadServiceSettings</tabstop>
  <tabstop>buttonSaveServiceSettings</tabstop>
  <tabstop>editServiceHomeDir</tabstop>
  <tabstop>buttonBrowseServiceHomeDir</tabstop>
  <tabstop>treeWidgetLog</tabstop>
 </tabstops>
 <resources>
  <include location="gex_ftp.qrc" />
 </resources>
 <connections/>
</ui>
